src/components/SVGIcon.vue

```diff
@@ -15,7 +15,7 @@
     xmlns="http://www.w3.org/2000/svg"
     xmlns:xlink="http://www.w3.org/1999/xlink"
   >
-    <use v-if="themeOverrideURL" :href="`${themeOverrideURL}#${themeId}`" />
+    <use v-if="themeOverrideURL" :href="`${themeOverrideURL}#${themeId}`" width="100%" height="100%" />
```

Comment:
Am I understanding correctly that this is only needed in the scenario where a referenced `<svg>` specifies its own explicit `width` or `height` attribute? Just trying to make sure I understand the scenario where this is breaking at the moment.

Reply:
Yep, exactly then.